linq4j/src/main/java/org/apache/calcite/linq4j/tree/Expressions.java

```diff
@@ -86,6 +86,14 @@ public static BinaryExpression add(Expression left, Expression right,
     throw Extensions.todo();
   }
 
+  /**
+   * Creates a BinaryExpression that represents an arithmetic
+   * mod operation that does not have overflow checking.
+   */
+  public static BinaryExpression mod(Expression left, Expression right) {
+    return makeBinary(ExpressionType.Mod, left, right);
```

Comment:
Which code would use this method ?

Reply:
'Expressions.mod()'  is used in ExpressionTest.